mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -165,35 +169,73 @@ class MDocUpdater : MDocCommand
 
 	HashSet<string> forwardedTypes = new HashSet<string> ();
 
-	public static string droppedNamespace = string.Empty;
+	public static DroppedAssemblyInfo.List droppedAssemblyList = new DroppedAssemblyInfo.List ();
+	public static string currentDroppedNamespace = string.Empty;
```

Comment:
For this particular thing, no ... all of this functionality is for maintaining documentation from two frameworks in a single set of XML docs ... since there is much shared code in this case. I'd originally kept the changes to both mdoc and monodoc in separate commits to try to break it up in that way, but I've since squashed those into one per early feedback on this PR.

But yes, generally I prefer to make smaller PRs when possible and avoid big "releases" like this ... this just happened to be a big job with lots of internal changes required :/